### PR TITLE
chore(CI): bump node version

### PR DESCRIPTION
**Pull request by czosel**
Node 14 is EOL. @cseufert strictly speaking this is a breaking change - should we treat it as such?

### CODE REVIEW

**Comment by cseufert** [comment]
We are not yet breaking on node14 are we? Do we need to hard drop support for it, or should we wait until a dependency/feature can no longer support node 14?

**Comment by czosel** [comment]
Totally, we aren’t. Shall we leave node 14 in CI until it breaks then?

**Comment by cseufert** [comment]
Yes, I think we leave it, and once it breaks our next release will drop support for it.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -9,47 +9,47 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [16.x, 18.x, 20.x]
```
Comment (cseufert) [inline .github/workflows/test.yml:12]:
```suggestion
        node-version: [14.x, 16.x, 18.x, 20.x]
```

Keep node 14.x support for now

**Review by cseufert — CHANGES_REQUESTED** [review]

**Review by cseufert — APPROVED** [review]
LGTM

**Pull request merged**